examples/configs/cryo-tcp.yaml

```diff
@@ -1,15 +1,11 @@
-- tickit.core.components.device_simulation.DeviceSimulation:
-    adapters:
-    - tickit.devices.cryostream.cryostream.CryostreamAdapter: {}
-    device:
-      tickit.devices.cryostream.cryostream.Cryostream: {}
+- tickit.devices.cryostream.Cryostream:
```

Comment:
Did we ever decide on 
```yaml
- type:
   - params
```

vs. 

```yaml
- name:
  - type: type
  - params
```

?

Reply:
We went for the first